src/System.Runtime.WindowsRuntime/src/System.Runtime.WindowsRuntime.builds

```diff
@@ -9,6 +9,10 @@
       <OSGroup>Windows_NT</OSGroup>
       <TargetGroup>netcore50aot</TargetGroup>
```

Comment:
You should delete this target, and delete the files that are moved to corert. It is what we are doing elsewhere.